% Template for PLoS
% Version 3.4 January 2017
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended 
% to minimize problems and delays during our production 
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % % 
%
% Once your paper is accepted for publication, 
% PLEASE REMOVE ALL TRACKED CHANGES in this file 
% and leave only the final text of your manuscript. 
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that 
% no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file. 
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission. 
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}
\usepackage{gensymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
% \bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}

% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\setlength{\headheight}{27.023pt}
\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\sf PLOS}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION


\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Consistent differences in fitness traits across multiple generations of Olympia oysters} % Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Katherine Silliman\textsuperscript{1*},
Tynan K. Bowyer\textsuperscript{1},
Steven B. Roberts\textsuperscript{2}
\\
\bigskip
\textbf{1} Ecology and Evolution, University of Chicago, Chicago, IL, U.S.A.
\\
\textbf{2} School of Aquatic and Fishery Sciences, University of Washington, Seattle, WA, U.S.A.
\\
\bigskip

% Use the asterisk to denote corresponding authorship and provide email address in note below.
* ksilliman@uchicago.edu

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Adaptive evolution and plasticity are two mechanisms that facilitate phenotypic differences between populations living in different environments. Understanding which mechanism underlies variation in fitness-related traits is a crucial step in designing conservation and restoration management strategies for taxa at risk from anthropogenic stressors. Olympia oysters (\textit{Ostrea lurida}) have received considerable attention with regard to restoration, however there is limited information on adaptive population structure. Using oysters raised under common conditions for up to two generations (F1s and F2s), we tested for evidence of divergence in reproduction, larval growth, and juvenile growth among three populations in Puget Sound, Washington. We found the population with the fastest growth rate also exhibited delayed and reduced reproductive activity, indicating a potential adaptive trade-off. Our results corroborate and extend upon a previous reciprocal transplant study on F1 oysters from the same populations, indicating that variation in growth rate and differences in reproductive timing are consistent across both natural and laboratory environments and have a strongly heritable component that cannot be entirely attributed to plasticity. 

\linenumbers
\section*{Introduction}

Natural environments exhibit spatial heterogeneity in both abiotic and biotic factors, oftentimes driving populations to evolve traits that confer a fitness advantage in their native habitat over foreign genotypes~\cite{Kawecki2004-qo}. This process of local adaptation can be opposed by homogenizing gene flow from dispersal, a significant factor for marine species with an extended planktonic dispersal phase~\cite{Grosberg2001-tv}. However, if the scale of dispersal extends across strong selective gradients, adaptive population divergence can still occur through phenotype-environment mismatch- where strong purifying selection occurs each generation following dispersal~\cite{Marshall2008-et,Schmidt2001-ex}. Characterizing the spatial scale and magnitude of adaptive divergence is a crucial step in designing conservation and restoration management strategies for taxa at risk from anthropogenic stressors~\cite{Baums2008-od,Carroll2014-ts}.\par
Conclusively demonstrating adaptive divergence is complicated by phenotypic plasticity, where individuals adjust their phenotype according to the conditions they experience\cite{West-Eberhard2003-ja}, which may confound inferences of local adaptation\cite{Kawecki2004-qo,Teplitsky2008-fe}. Phenotypic plasticity is widespread in marine species\cite{Conover2006-tk,Pepin1991-ja,Padilla2013-th}, and for marine invertebrates the most common trigger for plasticity appears to be the abiotic environment\cite{Padilla2013-th}. Organisms can be raised their entire lives in common conditions in order to minimize the effects of phenotypic plasticity. However, this approach may fail with strong transgenerational plasticity (TGP) - defined here as when the environment or phenotype of the parent affects the phenotype of the offspring\cite{Kawecki2004-qo,Guillaume2016-vu}. The ideal experimental design to distinguish TGP from genetic change involves raising and breeding individuals for at least two generations in a common setting\cite{Kawecki2004-qo}, although TGP has persisted for more than two generations in some laboratory studies\cite{Hercus2000-ai}. In a recent review of experimental evidence for local adaptation in marine invertebrates, only 11 out of 59 studies utilized 2 or more generations\cite{Sanford2011-fy}. Distinguishing between plastic responses and adaptive evolution to the environment is key to understanding the potential for marine species to acclimate or adapt to changing environmental conditions\cite{Salinas2013-ry}\par
Marine molluscs, and bivalves in particular, constitute some of our most economically and ecologically important marine invertebrates. Like many other marine invertebrates, they exhibit complex life cycles which include both planktonic larval stages as well as benthic juvenile and adult stages. The larval stage of many marine molluscs has been shown to be particularly sensitive to risks from ocean acidification and warming\cite{Parker2013-xg,Byrne2011-hk}, resulting in an increasing need to understand the relative importance of adaptive and plastic processes in shaping phenotypic variation. Evidence that TGP might be common for marine molluscs is growing (see \cite{Ross2016-en} for a thorough review), however only a handful of studies investigating adaptive differentiation in this clade compared organisms raised in common conditions for at least 2 generations, and of those most involved Gastropoda (see \cite{Kuo2009-co,Palmer1994-nw,Sanford2010-df,Dittman1998-xm,Bible2016-rb}).\par 

The Olympia oyster (\textit{Ostrea lurida}) is an estuarine species natively distributed from the central coast of Canada to Baja California. As a rhythmic consecutive hermaphrodite, spawning events in \textit{O. lurida} are thought to be synchronized between males and females based on environmental cues of temperature and seasonality\cite{Coe1932-nq}. Considered ecosystem engineers in estuaries, they provide structured habitat, remove suspended sediments, and limit eutrophication\cite{Ruesink2005-lt}. Following devastating commercial exploitation in the early 20th century, recovery of Olympia oyster populations has been stifled by other anthropogenic threats (water quality issues, habitat loss, and possibly ocean acidification)\cite{Blake2012-ln,Hettinger2013-od}. There is increasing political and economic pressure to restore abundance and recover ecosystem services offered by this species, which has spurred increasing interest in understanding the genetic, phenotypic, and epigenetic variation at both local and regional scales\cite{Camara2009-sn}.\par 
A recent study on Olympia oysters in central California provided evidence for spatial adaptive differentiation through a reciprocal transplant experiment with first-generation laboratory-reared (F1) oysters. The authors also found suggestive evidence of population-level differences in low salinity tolerance in second-generation, laboratory-reared (F2) oysters, and hypothesized adaptive divergence may occur in Olympia oysters over distances as short as 20-100 km\cite{Bible2016-rb}. In Puget Sound from 2013-2014, Heare \textit{et. al} 2017\cite{Heare2017-uv} conducted a reciprocal transplant experiment with F1 Olympia oysters from three distinct populations (Dabob Bay, Oyster Bay, and Fidalgo Bay). Variation in survival, growth rate, and reproductive activity was observed among populations and the four transplant sites. In particular, oysters from Fidalgo Bay had faster growth rates and reduced or delayed reproductive activity at most sites, while oysters from Dabob Bay had better survival yet slower growth rates, indicating potential adaptive trade-offs\cite{Heare2017-uv}.\par 
Although both of these studies controlled for environmental exposure of broodstock for up to 5 months prior to producing F1 oysters, they could not entirely rule out TGP as a causal factor. To test if phenotypic differences observed during the Puget Sound reciprocal transplant are consistent across generations, we conducted a common garden study in the summer of 2015 on F1 and F2 oysters derived from the same three Puget Sound populations. Concerns over TGP would be mitigated if F2 animals showed similar interpopulation differences as the F1s, indicating that these differences are heritable. Our study further developed on previous work by investigating growth rate at both the larval and juvenile life stages. 

\section*{Materials and methods}
\subsection*{Broodstock}

% For figure citations, please use "Fig" instead of "Figure".
Adult oysters were collected from three locations in Puget Sound, Washington; Fidalgo Bay (N 48.478252, W 122.574845), Oyster Bay (N 47.131465, W 123.021450), Dabob Bay (N 47.850948, W 122.805694) during November and December 2012. Oysters were held for 5 months in common conditions in Port Gamble, Washington and spawned in June 2013. Unlike many other oyster genera that broadcast spawn both eggs and sperm (e.g. \textit{Crassostrea}), Olympia oyster females are fertilized with spermatozeugmata ('sperm packets') from the water column and brood larvae for approximately 10-12 days. After being released into the water column, larvae are planktonic for approximately two weeks before attaching to a hard substrate ('settlement'). To ensure genetic diversity, each population was allowed to spawn in 24 separate groups of 20-25 oysters. Larvae produced from each population were reared in tanks based on spawning group, settled on very small pieces of oyster shell, then fed ad libitum. In August 2013, 480 juvenile oysters (5-10 mm) from each source population were outplanted at Clam Bay located in central Puget Sound (N 47.571839, W 122.550813), a different site than any of the source populations. For the purposes of this study we will refer to the cohort outplanted at Clam Bay as F1s. Reproductive and growth characteristics of F1 oysters at Clam Bay have been described by Heare \textit{et. al.} 2017. In June 2015, F1 oysters were moved into NOAA's Kenneth K. Chew Center for Shellfish Research and Restoration in Manchester, WA and maintained in mesh bags suspended in separate 18.9 L buckets with a diet of mixed live algae in flowing seawater.

\subsection*{Larval Rearing and Quantifying Reproductive Activity}
Spawning of broodstock (F1) was induced by elevating temperature to 18-20 \degree{C} in May 2015 and maintaining algae supplementation at 60,000-80,000 cells/mL. To ensure genetic diversity, each population (Fidalgo Bay = 101, Oyster Bay = 100, Dabob Bay = 94) was divided into 5 groups of 16-21 oysters. In addition to ensuring that multiple females were involved in reproduction, these replicate groups allowed us to statistically test population-level differences in reproductive activity. Larval release was checked and quantified every one to three days with larvae filtered (100 $\mu$M) and counted with triplicate drop counts. A cohort of these F2 larvae were used in the larval growth trials with the remainder raised in tanks (100 L) for the juvenile growth trial.

\subsection*{Experimental Set-up: Larvae Growth}
To investigate differences in growth rate among populations at the larval stage, we set up a larval growth rate experiment starting on a day when all three populations were producing at least 25,000 total larvae across multiple buckets (i.e. multiple females). Larvae were pooled by population and mixed well in 1 L beakers. The concentration of larvae was estimated using triplicate drop counts, and 900 larvae per population were added to each of three replicate plastic beakers (1 L) in order to account for random effects due to the beaker environment. These beakers were each outfitted with a "silo", a 7.62cm (3") diameter section of conditioned PVC pipe covered in 100 $\mu$M mesh on the bottom. Larvae remained in the silo, which allowed for daily water changes by lifting up the silo and inserting into a new beaker with premixed fresh seawater (800 ml) containing a 50/50 mix of live T. isochrysis and diatom algae (final concentration ~60,000-80,000 cells/mL). Approximately 20 larvae were sampled haphazardly by pipette from the initial larval pools at Day 0 and each replicate at Day 7 and Day 14 of the experiment, fixed in 10\% buffered formalin, and photographed under a microscope for analysis using ImageJ v1.51\cite{Schneider2012-js} software to determine shell area and length. We report results based on shell length, although shell area gave qualitatively similar results (not shown).

\subsection*{Experimental Set-up: Juvenile Growth}
Larvae from each population greater than 224 $\mu$M (n=30,000) were moved to new tanks (100 L) where air was bubbled to maintain oxygen levels and stimulate water movement. These tanks were lined with PVC tiles (10x10cm) roughed on one side using coarse sandpaper to promote oyster settlement. After four weeks, settled oysters were culled to fewer than 30 oysters/tile to avoid overgrowth interactions (Fidalgo tiles = 10, Oyster Bay tiles = 7, Dabob tiles = 8). Tiles were randomized and attached to four protected outplanting trays that were suspended from the dock at NOAA Manchester Research Station (depth = 6m). Photos were taken of oysters on tiles prior to outplanting (Day 0), after 48 days, and after 68 days for analysis using ImageJ software to determine shell area. 

\subsection*{Analysis}
To compare differences in reproductive activity among populations in the F1 generation, the five separate broodstock buckets per population were considered as independent replicates. The number of larvae released in each bucket on each sampling day was normalized by the number of adults in that bucket. To determine if there was a difference in total reproductive output among populations, the cumulative number of larvae produced throughout the 7 weeks for each bucket was analyzed using a one-way analysis of variance (ANOVA; R base) with post hoc Tukey's tests. In order to determine if populations differed in their timing of reproductive activity, one-way ANOVAs were also conducted on the number of days until the first observed larval release in each bucket after spawning conditions were induced, as well as the number of days until spawning peaked in each bucket.\par 
Linear mixed models (LMMs) were used to measure the effect of source population on oyster growth using the R package \textit{lme4}\cite{Bates2015-ky}. For the larval growth experiment, randomly selected live larvae were measured on Day 0, Day 7 and Day 14 with 10-12 larvae (mean=11.8, std=0.62) larvae per replicate. Dead oysters are easily distinguished from living oysters by having an entirely clear protoshell and no observable tissue. As \textit{O. lurida} larvae grow prior to maternal release, a one-way ANOVA was used to test if size varied among populations on Day 0 prior to separating out into replicate beakers. For the LMM analysis, population was a fixed effect and replicate beaker was a random effect. Prior to running the LMM, size distributions were tested for normality using the Shapiro-Wilkes test with the stats R package. Significance of the LMM results was established using a Likelihood Ratio Test against a null model based only on random effects. Shell length was compared at each time point using pairwise t-tests with a Bonferroni correction for multiple testing. \par 
For the juvenile experiment, growth was tracked on an individual basis (growth rate = $\Delta$area/\# days). All oysters that were alive (as determined by a healthy shell color and response to prodding) and not extending $>$50\% off the tile at each time point were measured for shell area. For the linear mixed model, population was a fixed effect and tray containing the tile was a random effect. Growth rate was natural log-transformed based on indications of non-normal distributions from the Shapiro-Wilkes test. Pairwise comparisons for populations at each time point were performed with the Nemenyi post hoc test using information from the Kruskal-Wallis test (\textit{PMCMR} package)\cite{Pohlert2014-lr}.

\subsection*{Data and Code Availability}
The datasets generated during current study are available on figshare [following publication acceptance]. Reproducible R Markdown notebooks detailing the code used for statistical analyses can be found at www.github.com/ksil91/PS-Oly-Larvae-Growth.

\section*{Results}
\subsection*{Reproductive Activity}
The timing and quantity of larvae produced varied across the three populations (Fig~\ref{fig1}). The cumulative number of larvae produced over a 7 week period differed significantly among population (one-way ANOVA, Df = 2, F = 4.174, \textit{p} = 0.0421), with F1 oysters from Fidalgo producing the fewest (Fig~\ref{fig2}a). Combined across all replicates, Oyster Bay oysters produced 2.7 million larvae, Dabob oysters produced 2.4 million, and Fidalgo oysters produced 1.1 million.

\begin{figure}[!h]
\includegraphics[width=0.7\linewidth]{overlay_larvae_timing_PS2015_color_percap.jpeg}
\caption{{\bf Reproductive activity through time in F1 oysters from three Puget Sound populations.}
The left axis measures the number of larvae released on each sampling day, summed up across replicates and normalized by the number of adult oysters in each replicate. The right axis measures the cumulative number of larvae produced through time, normalized by the number of adult oysters in each population.}
\label{fig1}
\end{figure}

\begin{figure}[!h]
\includegraphics[width=0.7\linewidth]{Cum_1stDay_Comb_PS2015_color.jpeg}
\caption{{\bf Cumulative larvae released and timing of first larvae release.}
A: Cumulative number of larvae released within each replicate bucket over 7 weeks, normalized by the number of oysters in each bucket. Oysters from Fidalgo produced significantly fewer than those from Oyster Bay (Tukey post hoc test, \textit{p} = 0.0499) and fewer, although not significantly, than those from Dabob (Tukey post hoc test, \textit{p} = 0.0954). B: Calendar day of first observed larvae release. Fidalgo oysters released larvae 10 days later than Oyster Bay oysters (Tukey post hoc test, \textit{p} = 0.0434) and 7 days later than Dabob oysters on average, although this was not statistically significant (Tukey post hoc test, \textit{p} = 0.156).}
\label{fig2}
\end{figure}

The onset of larval release also differed significantly among populations (one-way ANOVA, Df = 2, F = 4.033, \textit{p} = 0.0457), with Fidalgo oysters exhibiting delayed reproduction compared to the other populations and much higher variance in the date of initial larval release (Fig ~\ref{fig2}b). The timing of peak larval production did not vary significantly among populations (ANOVA, Df = 2, F = 0.097, \textit{p} = 0.908).

\subsection*{Larval Growth Experiment}
Significant differences in larvae size were not detected among populations on Day 0 of the larval growth experiment (one-way ANOVA, Df = 2, F = 0.939, \textit{p} =0.401). By Day 7, size varied significantly among populations (linear mixed model(LMM), \textit{p} = 7.939e\textsuperscript{-5}). Fidalgo larvae were 8\% larger than Dabob larvae (t-test, \textit{p} = 1.8e\textsuperscript{-6}) and 6\% larger than Oyster Bay larvae (t-test, \textit{p} = 0.00026). After 14 days, size still varied significantly among populations (LMM, \textit{p} = 0.03573), but only the comparison between Fidalgo and Dabob larvae remained significant (9\% larger; t-test, \textit{p} = 0.0017) (Fig ~\ref{fig3}).

\begin{figure}[!h]
\includegraphics[width=0.7\linewidth]{Larvae_Length_Line_PS215_color.jpeg}
\caption{{\bf Larval shell length of F2 oysters from three populations over 14 days.}
Data are means across replicates += s.e.m. Size varied significantly among populations at Day 7 (LMM,\textit{p} = 7.939e\textsuperscript{-5}) and Day 14 (LMM, \textit{p} = 0.03573).}
\label{fig3}
\end{figure}

\subsection*{Juvenile Growth Experiment}
Significant differences in juvenile shell area at Day 0 were not detected (one-way ANOVA, Df = 2, F = 0.483, \textit{p} = 0.617). Growth rate between Day 0 and Day 48 diverged among populations (LMM, \textit{p} = 0.02236). Fidalgo oysters grew 46\% faster than Dabob oysters (Kruskal-Wallis post hoc test, \textit{p} = 0.011), but all other pairwise comparisons were not significant. Between 48 and 68 days, shell growth continued to differ among populations (LMM, \textit{p} = 0.0012). Dabob oysters grew slower over this time period than Fidalgo oysters (Kruskal-Wallis post hoc test, \textit{p} = 0.0027) and tended to grow slower than Oyster Bay oysters (Kruskal-Wallis post hoc test, \textit{p} = 0.0806)(Fig ~\ref{fig4}).

\begin{figure}[!h]
\includegraphics[width=0.7\linewidth]{Juvenile_Growth_Bar_PS215_color}
\caption{{\bf Juvenile shell area growth rate of F2 oysters from three populations over 9 weeks.}
Juvenile shell growth of F2 oysters (growth rate = $\Delta$area / \# days). Growth rate between Day 0 - Day 48 differed significantly among populations (LMM, \textit{p} = 0.02236) as well as between Day 48 - Day 68 (LMM, \textit{p} = 0.0012).}
\label{fig4}
\end{figure}

\section*{Discussion}
The results presented here provide evidence for a strong heritable component underlying phenotypic variation in growth and reproduction among three populations of Olympia oysters in Puget Sound, WA. By building on the Heare \textit{et al.} 2017 study and showing consistent interpopulation differences between both first-generation (F1) and second-generation (F2), commonly reared oysters, this study indicated that inter-population variation at these fitness-related traits was not likely shaped by transgenerational plasticity. Persistent population differences in growth rate and reproductive timing across several generations have also been documented for selectively-bred strains of the eastern oyster, \textit{Crassostrea virginica}\cite{Dittman1998-xm,Barber1991-og}, suggesting these traits may have limited plasticity in bivalves.\par
Reproductive activity was characterized for the adult F1 oysters, with Oyster Bay oysters producing the most larvae over the course of 7 weeks and the Fidalgo population demonstrating delayed reproductive activity. These results are in accord with the reciprocal transplant study by Heare \textit{et al.}. By showing consistent reproduction patterns in both variable natural environments and controlled laboratory conditions, we demonstrated that reproductive timing in this species is not exclusively mediated by environmental conditions, but is also under genetic and/or epigenetic control. This result is in concordance with recent field studies indicating that nearby populations of Olympia oysters vary in temperature thresholds for reproduction\cite{Barber2016-ws,Seale2009-uw}. The development of asynchronous reproduction on such small spatial scales has major implications for limiting gene flow and contributing to population divergence\cite{Palumbi1994-rv}.\par
Fidalgo oysters exhibited the fastest growth during both the F2 larval and juvenile stages, while Dabob oysters exhibited the slowest growth, resulting in a significantly smaller size than Fidalgo oysters and smaller (although not significantly) than Oyster Bay oysters. This result is consistent with findings in F1 oysters by Heare \textit{et al.}, indicating a fixed underpinning for growth rate differences that manifests during both the larval and juvenile life stages. Interestingly, Fidalgo oysters also exhibited severely reduced and delayed reproductive activity. One explanation for this is an adaptive trade-off in energy allocation\cite{Perrin1993-wa,Folkvord2014-jj}, where Fidalgo oysters are devoting more energy to shell growth and less to gonad development. Heare \textit{et al.} also observed a potential adaptive trade-off in their reciprocal transplant study, where the population with slowest growth also had the highest survival across sites. Further investigation is required to fully resolve the link between growth, reproductive activity, and survival.\par
Environmental conditions in temperature, freshwater input, primary productivity, and anthropogenic waste effluent are known to vary among these sites\cite{Heare2017-uv}, supporting the possibility of selection driving the phenotypic variation observed in this study. In particular, up to 10 fold less chlorophyll a has been observed in Fidalgo Bay compared to the other sites. The proposed trade-off in energy allocation for this population could be driven by selection from such environmental differences in the quantity and timing of food supply\cite{Heino1999-fb,Pontarp2015-oc}. As mentioned previously, selection could act through local adaptation or phenotype-environment mismatch, depending on the spatial scale of dispersal relative to the scale of environmental heterogeneity. Although dispersal information is not currently available for Olympia oyster larvae in Puget Sound, estimates using chemical fingerprinting in Southern California identified considerable larval exchange among estuaries separated by up to 75 km\cite{Carson2010-xi}, which encompasses the distance between the populations used in this study. Further research using both chemical fingerprinting and genetic approaches are required to understand patterns of dispersal within Puget Sound in order to elucidate the mechanisms of adaptive population divergence.\par
Our results have implications for ongoing restoration efforts attempting to rebuild Olympia oyster populations. Current protocols for hatchery-based Olympia oyster restoration in Puget Sound involve using wild broodstock to produce hatchery-raised juveniles for outplanting in the same source population as the broodstock\cite{Blake2012-ln}, as opposed to combining broodstock from separate populations to increase genetic diversity. These efforts have focused on multiple sites in Puget Sound, including those covered in this study. Our results support this practice, with some caveats: 1) If local adaptation is indeed driving the observed phenotypic population structure, populations may be adapted to historical, rather than current, conditions. The convention of replenishing populations with only local broodstock sources may not provide the genetic diversity required to adapt in the face of rapid anthropogenic-induced environmental changes\cite{Jones2013-pe}. 2) Hatchery conditions vary dramatically from natural summer spawning conditions, and therefore artificial selective pressures are likely at play in the production of both the F1 and F2 oysters used in this study\cite{McClure2008-al}. However, the observed differences in size among populations are corroborated by field observations of wild oysters (Brady Blake, WA Dept of Fish and Wildlife,7-10-2017, pers comm).\par

\section*{Conclusion}
Despite the relatively close proximity of these populations and the potential for high gene flow, we observed significant phenotypic differences in fitness-related traits, even after multiple generations in the same environment. Of note, the population with the fastest growth rate also exhibited delayed and reduced reproductive activity, indicating a potential adaptive trade-off. Our results suggest these trait differences are heritable, as opposed to driven by transgenerational phenotypic plasticity, which has considerable implications for ongoing restoration efforts in this species. Further research is now required to understand the mechanisms of inheritance underlying these observed differences, whether they be genetic or epigenetic. 

\section*{Acknowledgments}
We thank Puget Sound Restoration Fund for providing facilities, equipment, and help with animal husbandry during these experiments and the NOAA Manchester Research Station for allowing us to deploy equipment off of their dock. We also thank Brent Vadopalas, Catherine Pfister, and Jay Dimond for helpful comments on earlier drafts. This material is based upon work supported by the National Science Foundation Graduate Research Fellowship under Grant No. 1545870. This work was funded in part by a grant from Washington Sea Grant, University of Washington, pursuant to National Oceanic and Atmospheric Administration Award No. NA10OAR4170057. The views expressed herein are those of the authors and do not necessarily reflect the views of NOAA or any of its sub-agencies.

\nolinenumbers
\begin{thebibliography}{10}

\bibitem{Kawecki2004-qo}
Kawecki TJ, Ebert D.
\newblock Conceptual issues in local adaptation.
\newblock Ecology Letters. 2004;7(12):1225--1241.

\bibitem{Grosberg2001-tv}
Grosberg R, Cunningham CW.
\newblock Genetic structure in the sea.
\newblock In: Marine community ecology. Sunderland, MA: Sinauer; 2001. p.
  61--84.

\bibitem{Marshall2008-et}
Marshall DJ.
\newblock Transgenerational plasticity in the sea: context-dependent maternal
  effects across the life history.
\newblock Ecology. 2008;89(2):418--427.

\bibitem{Schmidt2001-ex}
Schmidt PS, Rand DM.
\newblock Adaptive maintenance of genetic polymorphism in an intertidal
  barnacle: habitat- and life-stage-specific survivorship of Mpi genotypes.
\newblock Evolution. 2001;55(7):1336--1344.

\bibitem{Baums2008-od}
Baums IB.
\newblock A restoration genetics guide for coral reef conservation.
\newblock Molecular Ecology. 2008;17(12):2796--2811.

\bibitem{Carroll2014-ts}
Carroll SP, Jorgensen PS, Kinnison MT, Bergstrom CT, Denison RF, Gluckman P,
  et~al.
\newblock Applying evolutionary biology to address global challenges.
\newblock Science. 2014;346(6207).

\bibitem{West-Eberhard2003-ja}
West-Eberhard MJ.
\newblock Developmental plasticity and evolution.
\newblock Oxford University Press; 2003.

\bibitem{Teplitsky2008-fe}
Teplitsky C, Mills JA, Alho JS, Yarrall JW, Meril{\"a} J.
\newblock Bergmann's rule and climate change revisited: disentangling
  environmental and genetic responses in a wild bird population.
\newblock Proceedings of the National Academy of Sciences of the USA. 2008;105(36):13492--13496.

\bibitem{Conover2006-tk}
Conover DO, Clarke LM, Munch SB, Wagner GN.
\newblock Spatial and temporal scales of adaptive divergence in marine fishes
  and the implications for conservation.
\newblock Journal of Fish Biology. 2006;69:21--47.

\bibitem{Pepin1991-ja}
Pepin P.
\newblock Effect of temperature and size on development, mortality, and
  survival rates of the pelagic early life history stages of marine fish.
\newblock Canadian Journal of Fisheries and Aquatic Sciences. 1991;48(3):503--518.

\bibitem{Padilla2013-th}
Padilla DK, Savedo MM.
\newblock A systematic review of phenotypic plasticity in marine invertebrate
  and plant systems.
\newblock Advances in Marine Biology. 2013;65:67--94.

\bibitem{Guillaume2016-vu}
Guillaume AS, Monro K, Marshall DJ.
\newblock Transgenerational plasticity and environmental stress: do paternal
  effects act as a conduit or a buffer?
\newblock Functional Ecology. 2016;30(7):1175--1184.

\bibitem{Hercus2000-ai}
Hercus MJ, Hoffmann AA.
\newblock Maternal and grandmaternal age influence offspring fitness in
  \textit{Drosophila}.
\newblock Proceedings of the Royal Society of London: Biological Sciences. 2000;267(1457):2105--2110.

\bibitem{Sanford2011-fy}
Sanford E, Kelly MW.
\newblock Local adaptation in marine invertebrates.
\newblock Annual Review of Marine Science. 2011;3:509--535.

\bibitem{Salinas2013-ry}
Salinas S, Brown SC, Mangel M, Munch SB.
\newblock Non-genetic inheritance and changing environments.
\newblock Non-Genetic Inheritance. 2013;1:38-50.

\bibitem{Parker2013-xg}
Parker LM, Ross PM, O'Connor WA, P{\"o}rtner HO, Scanes E, Wright JM.
\newblock Predicting the response of molluscs to the impact of ocean
  acidification.
\newblock Biology. 2013;2(2):651--692.

\bibitem{Byrne2011-hk}
Byrne M.
\newblock Impact of ocean warming and ocean acidification on marine
  invertebrate life history stages.
\newblock In: Oceanography and Marine Biology. Oceanography and Marine Biology
  - An Annual Review. CRC Press; 2011.

\bibitem{Ross2016-en}
Ross PM, Parker L, Byrne M.
\newblock Transgenerational responses of molluscs and echinoderms to changing
  ocean conditions.
\newblock ICES Journal of Marine Science. 2016;73(3):537--549.

\bibitem{Kuo2009-co}
Kuo ESL, Sanford E.
\newblock Geographic variation in the upper thermal limits of an intertidal
  snail: Implications for climate envelope models.
\newblock Marine Ecology Progress Series. 2009;388:137--146.

\bibitem{Palmer1994-nw}
Palmer RA.
\newblock Temperature sensitivity, rate of development, and time to maturity:
  Geographic variation in laboratory-reared \textit{Nucella} and a
  cross-phyletic overview.
\newblock In: {Wilson, W H Stricker, S}, editor. Reproduction and development
  of marine invertebrates. Johns Hopkins University Press; 1994. p. 177--194.

\bibitem{Sanford2010-df}
Sanford E, Worth DJ.
\newblock Local adaptation along a continuous coastline: prey recruitment
  drives differentiation in a predatory snail.
\newblock Ecology. 2010;91(3):891--901.

\bibitem{Dittman1998-xm}
Dittman D, Ford SE, Haskin HH.
\newblock Growth patterns in oysters, \textit{Crassostrea virginica}, from
  different estuaries.
\newblock Marine Biology. 1998;132(3):461--469.

\bibitem{Bible2016-rb}
Bible JM, Sanford E.
\newblock Local adaptation in an estuarine foundation species: Implications for
  restoration.
\newblock Biological Conservation. 2016;193:95--102.

\bibitem{Coe1932-nq}
Coe WR.
\newblock Development of the gonads and the sequence of the sexual phases in
  the California oyster (\textit{Ostrea lurida}).
\newblock Scripps Institution of Oceanography. 1932;.

\bibitem{Ruesink2005-lt}
Ruesink JL, Lenihan HS, Trimble AC, Heiman KW, Micheli F, Byers JE, et~al.
\newblock Introduction of non-native oysters: ecosystem effects and restoration
  implications.
\newblock Annual Review of Ecology, Evolution, and Systematics. 2005;36(1):643--689.

\bibitem{Blake2012-ln}
Blake B, Bradbury A.
\newblock Washington Department of Fish and Wildlife plan for rebuilding
  Olympia oyster (\textit{Ostrea lurida}) populations in Puget Sound with a
  historical and contemporary overview.
\newblock Brinnon, WA: Washington Department of Fish and Wildlife; 2012.

\bibitem{Hettinger2013-od}
Hettinger A, Sanford E, Hill TM, Lenz EA, Russell AD, Gaylord B.
\newblock Larval carry-over effects from ocean acidification persist in the
  natural environment.
\newblock Global Change Biology. 2013;19(11):3317--3326.

\bibitem{Camara2009-sn}
Camara MD, Vadopalas B.
\newblock Genetic aspects of restoring Olympia oysters and other native
  bivalves: balancing the need for action, good intentions, and the risks of
  making things worse.
\newblock Journal of Shellfish Research. 2009;28(1):121--145.

\bibitem{Heare2017-uv}
Heare JE, Blake B, Davis JP, Vadopalas B, Roberts SB.
\newblock Evidence of \textit{Ostrea lurida} (Carpenter 1864) population
  structure in Puget Sound, {WA}.
\newblock Marine Ecology. 2017;.

\bibitem{Schneider2012-js}
Schneider CA, Rasband WS, Eliceiri KW.
\newblock {NIH} Image to {ImageJ}: 25 years of image analysis.
\newblock Nature Methods. 2012;9(7):671--675.

\bibitem{Bates2015-ky}
Bates D, M{\"a}chler M, Bolker B, Walker S.
\newblock Fitting linear mixed-effects models using lme4.
\newblock Journal of Statistical Software, Articles. 2015;67(1):1--48.

\bibitem{Pohlert2014-lr}
Pohlert T. The Pairwise Multiple Comparison of Mean Ranks package ({PMCMR});
  2014.

\bibitem{Barber1991-og}
Barber BJ, Ford SE, Wargo RN.
\newblock Genetic variation in the timing of gonadal maturation and spawning of
  the Eastern Oyster, \textit{Crassostrea virginica} (Gmelin).
\newblock Biological Bulletin. 1991;181(2):216--221.

\bibitem{Barber2016-ws}
Barber JS, Dexter JE, Grossman SK, Greiner CM, Mcardle JT.
\newblock Low temperature brooding of Olympia oysters (\textit{Ostrea lurida})
  in northern Puget Sound.
\newblock Journal of Shellfish Research. 2016;35(2):351--357.

\bibitem{Seale2009-uw}
Seale EM, Zacherl DC.
\newblock Seasonal settlement of Olympia oyster larvae, \textit{Ostrea lurida}
  Carpenter 1864 and its relationship to seawater temperature in two southern
  California estuaries.
\newblock Journal of Shellfish Research. 2009;28(1):113--120.

\bibitem{Palumbi1994-rv}
Palumbi SR.
\newblock Genetic divergence, reproductive isolation, and marine speciation.
\newblock Annual Review of Ecology and Systematics. 1994;25(1):547--572.

\bibitem{Perrin1993-wa}
Perrin N, Sibly RM, Nichols NK.
\newblock Optimal growth strategies when mortality and production rates are
  size-dependent.
\newblock Evolutionary Ecology. 1993;7(6):576--592.

\bibitem{Folkvord2014-jj}
Folkvord A, J{\o}rgensen C, Korsbrekke K, Nash RDM, Nilsen T, Skj{\ae}raasen
  JE.
\newblock Trade-offs between growth and reproduction in wild Atlantic cod.
\newblock Canadian Journal Fisheries and Aquatic Sciences. 2014;71(7):1106--1112.

\bibitem{Heino1999-fb}
{Heino}, {Kaitala}.
\newblock Evolution of resource allocation between growth and reproduction in
  animals with indeterminate growth.
\newblock Journal of Evolutionary Biology. 1999;12(3):423--429.

\bibitem{Pontarp2015-oc}
Pontarp M, Johansson J, Jonz{\'e}n N, Lundberg P.
\newblock Adaptation of timing of life history traits and population dynamic
  responses to climate change in spatially structured populations.
\newblock Evolutionary Ecology. 2015;29(4):565--579.

\bibitem{Carson2010-xi}
Carson HS.
\newblock Population connectivity of the Olympia oyster in southern California.
\newblock Limnology and Oceanography. 2010;55(1):134--148.

\bibitem{Jones2013-pe}
Jones TA.
\newblock When local isn't best.
\newblock Evolutionary Applications. 2013;6(7):1109--1118.

\bibitem{McClure2008-al}
McClure MM, Utter FM, Baldwin C, Carmichael RW, Hassemer PF, Howell PJ, et~al.
\newblock Evolutionary effects of alternative artificial propagation programs:
  implications for viability of endangered anadromous salmonids.
\newblock Evolutionary Applications. 2008;1(2):356--375.

\end{thebibliography}

\end{document}

